File: buildSrc/src/main/groovy/cn/mycommons/implementstransformplugin/Entry.java
package cn.mycommons.implementstransformplugin;

import cn.mycommons.modulebase.annotations.Implements;
import javassist.CtClass;

/**
 * Entry <br/>
 * Created by xiaqiulei on 2017-05-15.
 */
public class Entry {

    final Implements anImplements;
    final CtClass ctClass;

    public Entry(Implements anImplements, CtClass ctClass) {
        this.anImplements = anImplements;
        this.ctClass = ctClass;
    }

    @Override
    public String toString() {
        return "Entry{" +
                "anImplements=" + anImplements +
                ", ctClass=" + ctClass +
                '}';
    }
}

File: plugin/module-plugin-ksp/src/main/kotlin/cn/mycommons/module_plugin/ksp/Entry.kt
package cn.mycommons.module_plugin.ksp

import cn.mycommons.modulebase.annotations.Implements
import javassist.CtClass

/**
 * Entry <br></br>
 * Created by xiaqiulei on 2017-05-15.
 */
class Entry(
    val anImplements: Implements,
    val ctClass: CtClass,
) {

    override fun toString(): String {
        return "Entry{" +
                "anImplements=" + anImplements +
                ", ctClass=" + ctClass +
                '}'
    }
}